lang/stack/Portfile

```diff
@@ -84,12 +80,12 @@ post-extract {
     xinstall -m 0755 -W ${distpath} ./${distname}-bin ${workpath}/bin
 }
 
-# bootstrap binaries seg fault on 10.7 and older
-if {${os.platform} eq "darwin" && ${os.major} < 12} {
+# bootstrap binaries seg fault on OSX10.7 and older
+if { ${os.major} < 12 } {
```

Comment:
Please do not remove the os.platform check here, as the following os.major check is based on the presumption the platform is darwin.

Reply:
Chris! Is this you?

https://github.com/macports/macports-ports/commit/ab19012c66666246d451ef765bfca87461f130da#diff-cf93ab709a7a1e022a95a8b2fb855770e3bc2586c0b27dffbf982f668402fc9a

Reply:
Oops, I may have been a commit or two out of sync. I'll fix…

Reply:
I don’t know what you are doing to get git to to accept your commits that wipe out other changes but it sounds like in some way you are forcing the changes. Whatever it is you are doing in your workflow please review and change it as it is dangerous. This is not the first time this has happened.

in future before committing new PRs please first ensure the branch you are using is fully rebased against current master. The best way is to delete feature branches after they are merged, and create new ones off master each time you wish to make a PR.

```
git checkout master
git pull —rebase —autostash origin master
git checkout -b NewFeature
git add …..
git commit ….
git push essandess NewFeature
```